__tests__/api.test.js

```diff
@@ -0,0 +1,36 @@
+const request = require('supertest')
+const pool = require('../db/connection.js')
+const app = require('../index.js')
+const seed = require('../db/seeds/seed.js')
+const data = require('../db/data/test-data')
+
+
+
+beforeEach(() => {
+    return seed(data)
+})
+
+afterAll(() => {
+    return pool.end()
+});
+
+describe('GET /api/categories', () => {
+    test('gets categories ', () => {
+        return request(app)
+            .get('/api/categories')
+            .expect(200)
+            .then((res) => {
+                expect(res.body.length).toBe(4);
```

Comment:
categories again should be a key on the body object which will need to be updated here.